.travis.yml

```diff
@@ -118,18 +117,19 @@ script:
         fi;
     fi;
   - if [ "${TRAVIS_OS_NAME}" == "osx" ]; then
+      export USE_OPENGL_MOCK=1;
       if [ "${PY}" == "3" ]; then
          python3 setup.py build_ext --inplace;
-         echo -e "python3 -m nose.core kivy/tests";
+         python3 -m nose.core kivy/tests;
       else
          make;
-         echo -e "make test";
+         make test;
       fi;
     fi;
 
 
 after_success:
-  - if [ "${TRAVIS_OS_NAME}" == "osx" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
+  - if [ "${TRAVIS_OS_NAME}" == "os" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
```

Comment:
Yes, the block was "disabled" :-P because it's unfinished, it should upload to g drive instead of the server.